api/handlers.go

```diff
@@ -289,13 +303,42 @@ func (a *API) Unmount(w http.ResponseWriter, r *http.Request) {
 
 	volName := volConfig.String()
 
+	ut := &config.UseMount{
+		Volume:   volName,
+		Reason:   lock.ReasonMount,
+		Hostname: a.Hostname,
+	}
+
+	if !volConfig.Unlocked {
+		a.RemoveStopChan(volName)
```

Comment:
the stop channel (or `stopChan`) is the channel, when sent an empty struct, terminates the TTL refresher.

`AddStopChan` adds a stop channel to a registry which is keyed by volume name. any additional adds will terminate the old refreshers.
`RemoveStopChan` terminates the goroutine and removes the stop channel from the map.